packages/terra-list/src/ListSection.jsx

```diff
@@ -59,33 +76,126 @@ const ListSection = ({
   children,
   isCollapsed,
   isCollapsible,
+  isDraggable,
+  onDragEnd,
+  intl,
   ...customProps
 }) => {
-  let sectionItems;
-  if (!isCollapsible || !isCollapsed) {
-    sectionItems = children;
-  }
+  const [listItemNodes, setlistItemNodes] = useState(children);
+  let listNode = useRef();
+
+  useEffect(() => {
+    if (!isCollapsible || !isCollapsed) {
```

Comment:
Are you sure all cases are handled appropriately?

if isCollaspsible = true, isCollapsed = true, this will evaluate as false
if isCollaspsible = false, isCollapsed = true, this will evaluate as true
if isCollaspsible = false, isCollapsed = false, this will evaluate as true
if isCollaspsible = true, isCollapsed = false, this will evaluate as true

Reply:
Yes. This is the existing logic in terra-list component. Kept intact. Thanks